src/core/CellHeader/style.ts

```diff
@@ -76,8 +78,8 @@ export const StyledTableHeader = styled("th", {
       &:hover {
         color: ${active ? colors?.primary[500] : "black"};
 
-        & .MuiButtonBase-root {
-          color: ${active ? colors?.primary[500] : colors?.gray[500]};
+        & > .MuiButtonBase-root {
```

Comment:
What is the expected behavior here? I noticed when I remove the `>` a dropdown appears when I hover over the text, and when it is there the dropdown isn't there:
![Screen Shot 2022-11-10 at 10 14 54 AM](https://user-images.githubusercontent.com/36094225/201133444-64b52f0f-932b-492f-b376-1aba484bc505.png)
![Screen Shot 2022-11-10 at 10 15 06 AM](https://user-images.githubusercontent.com/36094225/201133447-ed594333-e51c-4fd5-bdbf-3c6e954c7f80.png)

Reply:
I'm not sure where this change came from. I don't see it in my commit history, but maybe I'm missing something. It's unintentional though. Do I need to make a new commit and rerun the tests to make this change?

Reply:
hmm maybe it was by mistake? the '>' is not on the master branch so lets remove it.